sys/include/net/dhcpv6/client.h

```diff
@@ -131,6 +138,15 @@ void dhcpv6_client_start(void);
 void dhcpv6_client_req_ia_pd(unsigned netif, unsigned pfx_len);
 /** @} */
 
+/**
+ * @brief   Configures the client to request non-temporary addresses for a network
+ *          interface from a server
```

Comment:
Maybe add a note here, that for WPAN meshes a DHCPv6 relay (which is not implemented yet) is required.

Reply:
I included your comment in a note in 35b111b5f1ca22bc1ddf3b29f9d0b4f6e32544d2, feel free to request any changes :)